src/main/scala/tilelink/Broadcast.scala

```diff
@@ -446,10 +651,13 @@ class TLBroadcastTracker(id: Int, lineBytes: Int, caches: Int, bufferless: Boole
     user   :<= io.in_a.bits.user
     echo   :<= io.in_a.bits.echo
     address := io.in_a.bits.address
+    /* we need to fetch from lower level at least. */
     count   := 1.U
   }
-
+  /* cache [[cacheOH]] by [[io.clearOH]] */
```

Comment:
Again I think this repeats the code without clarifying its intent. (I think its intent is to clear some of the pending bits identifying which caches still need to be probed).